src/Skeletor/Manager/ProviderManager.php

```diff
@@ -0,0 +1,89 @@
+<?php
+
+namespace Skeletor\Manager;
+
+use Skeletor\Frameworks\Framework;
+use Skeletor\Packages\Package;
+
+class ProviderManager extends Manager
+{
+    const PROVIDER_METHOD = 'getProviderClass';
+    const FACADE_METHOD = 'getFacadeClass';
+
+    /**
+     * @param Package $package
+     * @param Framework $activeFramework
+     */
+    public function configure(Package $package, Framework $activeFramework)
+    {
+        if ($activeFramework->getName() !== 'Laravel') {
+            $this->cli->red("Configure {$package->getInstallSlug()} manual for {$activeFramework->getName()}, we currently only support Laravel");
+            return;
+        }
+        $configFile = $activeFramework->getPath('appConfig');
+        $newContent = $this->getNewConfig($configFile, $package);
+        $this->projectFilesystem->update($configFile, $newContent);
+    }
+
+    /**
+     * @param string $configFile
+     * @param Package $package
+     * @return array|bool
+     */
+    public function getNewConfig(string $configFile, Package $package)
+    {
+        $state = null;
+        $appConfig = file($configFile);
+
+        foreach($appConfig as $key => $line) {
+            $cleanLine = trim(preg_replace('/[\t\s]+/', '', $line));
+
+            switch ($cleanLine) {
+                case "'providers'=>[":
+                    $state = self::PROVIDER_METHOD;
+                    break;
+                case "'aliases'=>[":
+                    $state = self::FACADE_METHOD;
+                    break;
+            }
+
+            if ($cleanLine === "]," && $state !== null) {
+                $previousLine = --$key;
+                $appConfig[$previousLine] .= $this->$state($package);
+                $state = null;
+            }
+        }
+
+        return $appConfig;
+    }
+
+    /**
+     * @param Package $package
+     * @return string
+     */
+    public function getFacadeClass(Package $package)
+    {
+        $facade = $package->getFacade();
+        if (isset($facade) === false) {
+            return;
+        }
+
+        [$alias, $facade] = explode('@', $package->getFacade());
+
```

Comment:
@ThijsBouwes Is it possible to rewrite this bit so it would also work on `7.0`? If we want everyone to use it it might hold people back if they have to upgrade their local PHP version to `7.1` if they haven't already.